app/services/idv/steps/verify_step.rb

```diff
@@ -11,6 +11,10 @@ def call
 
       def enqueue_job
         return if flow_session[verify_step_document_capture_session_uuid_key]
+        if pii_from_doc.nil?
+          mark_step_incomplete(:ssn)
+          return
+        end
```

Comment:
I drafted a comment here but it looks like it was updated / fixed in the meantime. Still, depending if we want a one-liner, could be relevant:

---

Curious about the `||` here: Looking at the implementation of `mark_step_incomplete`, I might assume it returns a truthy value, assuming the corresponding step exists, and never hit the `return`.

https://github.com/18F/identity-idp/blob/9a9fc8a5cee41492b40bb912772201c723e612f4/app/services/flow/base_step.rb#L26

Wonder if we could make an explicit (`nil`?) return value from `mark_step_incomplete`, and write this as:

```rb
return mark_step_incomplete(:ssn) if pii_from_doc.nil?
```

... with added advantage that it's clearer from the start of the line that this is an early return path.